### PR TITLE
New cotton baguette, crostini, chevre-chaud, bagel, and croissant foods for moffs

### PR DESCRIPTION
**Pull request by ArtisticRoomba**
## About the PR
Adds new foods, all made from new cotton dough slices and cotton dough rolls.
The cotton baguette, cotton crostini, cotton chevre-chaud, cotton bagel, and cotton croissant.

You can make them by following the recipes for regular dough, but use cotton dough instead.

This PR also changes sliced and rolled normal dough having more nutriment than its whole counterpart. This prevents you from duplicating nutriment simply by slicing the dough.

## Why / Balance
Was marked as a TODO. Poor moffs need more foods and I think this will give chefs a lot of opportunity to have fun and make some fine dishes for the fellow moffs.
## Technical details
Just YAML changes.

## Media
Content Client_L7qo0AxgR5

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

:cl: ArtisticRoomba, JuneSzalkowska
- add: New cotton dough slices and cotton dough rolls.
- add: New cotton baguette, cotton crostini, cotton chevre-chaud, cotton bagel, and cotton croissant recipes. You can make them by following the recipes for regular baguettes, bagels, etc, but use cotton dough instead.
- fix: Slicing dough into thirds now divides the nutriment value into thirds. Previously you could slice dough to triple your nutriment.

### CODE REVIEW

**Comment by Pumkin69** [comment]
yeah this is fire

**Comment by onesch** [comment]
lethimcook

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bagel.yml
Comment (SlamBamActionman) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bagel.yml:73]: To have parity with cotton bread, the food solution should be 50% nutriment, 50% cotton of the original amount. They're meant to be primarily for moffs after all!

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml
Comment (SlamBamActionman) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml:708]: Same with introducing fiber in the solutionmanager here. I would suggest only splitting the nutriment 50-50 (i.e. 3 nutriment, 3 fiber) and keep the extra reagents like salt the same.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml
Comment (SlamBamActionman) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml:757]: Samesies

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/misc.yml
Comment (SlamBamActionman) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/misc.yml:615]: Samesies

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/misc.yml
Comment (SlamBamActionman) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/misc.yml:799]: Samesies

File: Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml
Comment (SlamBamActionman) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml:452]: Samesies

**Review by SlamBamActionman — CHANGES_REQUESTED** [review]
Requires some changes to make the entities contain fiber. The sprites will also require art maintainer approval; there is a concern it may look moldy, but at the same time we already have moldy bread and that is a very noticeable green, so this might be fine.

**Comment by ArtisticRoomba** [comment]
I've addressed your review (lol good job me) as well as pushed some alternative sprites in commit https://github.com/space-wizards/space-station-14/pull/33508/commits/3eecac7d9eb375e7d256ec378dfa2c2e894177bd for the art directors to see if they don't like the 3 original sprites.

Let me know if the sprites are not up to par.

**Review by SlamBamActionman — APPROVED** [review]
Approving the code side, looks good! Sprites will need art maintainer approval.

**Review by UbaserB — DISMISSED** [review]
The raw croissant and chevre-chaud sprites need to look more unique since they're really close to their counterpart sprites right now, and the bagel/cooked croissant sprites shouldn't look moldly. You can probably fix it by giving a layer of cotton in the middle (like a bagel sandwich) or almost like icing on the top... and for the croissant you could probably add cotton underneath the slits(?), like where the flaps/layers of the croissant are. The other sprites look fine.

**Review by UbaserB — COMMENTED** [review]
Just make the raw croissant sprite and the cooked cotton croissant sprites line have the cotton part line up. You can go either direction here, with it being spotted or just on the inner parts... or both. Doesn't matter too much.

Sorry for not mentioning this earlier, but the rest of the sprites look fine to me.

**Comment by ArtisticRoomba** [comment]
I've added some new sprites @UbaserB. Let me know what you think.

image

**Review by Emisse — APPROVED** [review]

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml:714]:
```suggestion
```
inherited

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml:722]:
```suggestion
```
inherited

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml:724]:
```suggestion
```
inherited

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/bread.yml:781]:
```suggestion
    - Bread
    - Slice
    - ClothMade
```

File: Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml:475]: why do the 3 slices have more reagents than the whole dough had before?

Comment (ArtisticRoomba) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml:475]: All regular dough slices and dough ropes inherit FoodBakingBase which has 15 nutriment, which means that you can turn 15 nutriment dough into 3 dough ropes/slices worth 45 nutriment combined.

Slam requested that I halve the nutriment content and replace it with fiber so that the moths actually get food out of the food.

Do you want me to fix the normal and moff dough solution contents so they actually contain the correct amount for 3 parts?

Comment (slarticodefast) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml:475]: yeah, fix the normal one as well please

**Review by slarticodefast — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**
